pkg/tnf/reel/reel.go

```diff
@@ -214,6 +215,10 @@ func (r *Reel) Step(step *Step, handler Handler) error {
 				}
 				match, matchStatus := r.stripEmulatedPromptFromOutput(result.Match[0])
 				log.Debugf("command status: output=%s, match=%s, outputStatus=%d, matchStatus=%d", output, match, outputStatus, matchStatus)
+				// If the exit code is not zero, then fail the test
+				if outputStatus != 0 {
+					ginkgo.Fail(fmt.Sprintf("Failed because of non-zero exit status = %d", outputStatus))
+				}
```

Comment:
is this intended to be permanent fix or temporary ? if temporary, then that's fine.
Otherwise, this function should return an error, then deal with the error at upper layer.
we should not be adding ginkgo dependency here.

Reply:
fixed